hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -152,7 +155,7 @@ private void addStatsOfNoDataIncludedMaps(Map statsPerMap) {
         Collection<String> mapNames = proxyService.getDistributedObjectNames(SERVICE_NAME);
         for (String mapName : mapNames) {
             if (!statsPerMap.containsKey(mapName)) {
-                statsPerMap.put(mapName, EMPTY_LOCAL_MAP_STATS);
+                statsPerMap.put(mapName, initializeLocalMapStats(mapName));
```

Comment:
I wonder if this change could affect performance somehow? Before a single object was reused for empty maps and now it's allocated per each map.